dev/io.openliberty.grpc.1.0.internal.monitor/src/io/openliberty/grpc/internal/monitor/GrpcServerStats.java

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.grpc.internal.monitor;
+
+import com.ibm.websphere.monitor.meters.Counter;
+import com.ibm.websphere.monitor.meters.Meter;
+
+import io.openliberty.grpc.GrpcServerStatsMXBean;
+
+/**
+ * This is used to report gRPC Server related statistics. </br>
+ * Statistic reported:
+ * <ul>
+ * <li>Total number of RPCs started on the server.
+ * <li>Total number of RPCs completed on the server, regardless of success or
+ * failure.
+ * <li>TODO Histogram of response latency of RPCs handled by the server, in
+ * seconds.
+ * <li>Total number of stream messages received from the client.
+ * <li>Total number of stream messages sent by the server.
+ * </ul>
+ */
+public class GrpcServerStats extends Meter implements GrpcServerStatsMXBean {
+
+	private String appName;
+	private String serviceName;
+
+	private Counter receivedMsgCount;
+	private Counter sentMsgCount;
+	private Counter serverStarted;
+	private Counter serverHandled;
+
+	public GrpcServerStats(String aName, String sName) {
+		setAppName(aName);
+		setServiceName(sName);
+
+		serverStarted = new Counter();
+		serverStarted.setDescription("This shows total number of RPCs started on the server");
```

Comment:
should these descriptions be NLS translated?

Reply:
I don't think so. I don't see other strings in NLS files.